core/model/upgradeseries.go

```diff
@@ -10,6 +10,18 @@ import (
 // UpgradeSeriesStatus is the current status of a series upgrade for units
 type UpgradeSeriesStatus string
 
+var UpgradeSeriesStatusOrder map[UpgradeSeriesStatus]int = map[UpgradeSeriesStatus]int{
+	UpgradeSeriesNotStarted:       0,
+	UpgradeSeriesPrepareStarted:   1,
+	UpgradeSeriesPrepareRunning:   2,
+	UpgradeSeriesPrepareMachine:   3,
```

Comment:
Can we delete this altogether now that it is unused?

Reply:
A little foresight never hurt anyone. Yes `UpgradeSeriesPrepareMachine` is being removed. So we can remove it here with no harm done.